vhdbuilder/packer/components.json

```diff
@@ -83,7 +83,8 @@
       "amd64OnlyVersions": [],
       "multiArchVersions": [
         "v1.3.4-1",
-        "v1.4.1"
+        "v1.4.1",
```

Comment:
v1.4.1 is just added 4 days ago, so you want to replace with v1.4.2 directly? since v1.4.1 does not take effect in aks-rp, why not repace with v1.4.2 directly?